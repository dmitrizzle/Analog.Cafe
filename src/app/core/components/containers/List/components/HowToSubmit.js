// tools
import React from "react"

// components
import { Section, Article } from "../../../stateless/ArticleStyles"
import Figure from "../../../stateless/Picture"
import { LinkButton } from "../../../stateless/_controls/Button"

// render
export default () => {
  return (
    <Article>
      <Section plain>
        <h3>How to submit your work.</h3>
        <p>
          Analog.Cafe lets you create and edit your submission using a tool
          called <em>Composer</em>.
        </p>
        <p>
          It’s recommended that you use latest Google Chrome, Safari or Firefox
          browser on your desktop computer to compose your submission.
        </p>
        <p>
          We review and edit all works before publishing. Once you send us your
          first submission it will appear on this page. As with most
          publications, not all submissions can be accepted.
        </p>
        <Figure
          src="image-froth_1101010_20d7f3aee5134e109123d1f7e132185c.gif"
          nocaption
          alt="Analog.Cafe Composer in action"
        />
        <p>
          While composing your submission keep in mind that it needs to have a
          concise and descriptive title and at least one image. You can add text
          formatting to your entry (such as <strong>bold</strong> and{" "}
          <em>italics</em>), links headings, and quotes. You can also move the
          images around, choose to have them full-screen or float left for the
          desired effect.
        </p>
        <p>
          When inserting images into your post you may also choose one of the
          recommended photographs that show up in a grid when you click the “Add
          Image” button. They’ve been created by other Analog.Cafe contributors
          and selected by the editor. Should you choose to use one, both of you
          will be credited appropriately with links to your profiles. You will
          also be listed under “Collaborations” catogory on the website.
        </p>
        <Figure
          src="image-froth_1498127_rJE9bhJNG"
          nocaption
          alt="Instant Collaborations"
        />
        <p>
          If you would like to offer your images up for being displayed in other
          authors’ publications click “Yes” for{" "}
          <strong>“Instant Collaborations”</strong> when you are submitting your
          work and we may include it in the image suggestions grid for you.
        </p>
        <p>
          You may also edit your profile image, bio and a link to your website
          or social media page. Just click on “More…” in the naviagation bar to
          find what you need.
        </p>
        <LinkButton to={"/submit/compose"} branded>
          Submit Now
        </LinkButton>
      </Section>
    </Article>
  )
}
