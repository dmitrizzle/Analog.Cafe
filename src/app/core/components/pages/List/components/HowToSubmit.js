import React from "react"

import Submit from "../../../../../user/components/pages/Submit"

export default () => {
  return <Submit embed />
}
