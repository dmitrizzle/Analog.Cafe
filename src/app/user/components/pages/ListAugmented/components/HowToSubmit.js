import React from "react"

import ArticleSection from "../../../../../core/components/pages/Article/components/ArticleSection"
import Link from "../../../../../core/components/controls/Link"

export default () => {
  return (
    <ArticleSection>
      <p>
        If you love shooting film and have a story to share, Analog.Cafe is the
        place to publish it. Submissions are now accepted for essays and
        articles on or featuring film photography.{" "}
        <Link to="/submit">Details</Link>.
      </p>
      <p>
        This is the page where your active or published submissions will appear.
      </p>
    </ArticleSection>
  )
}
