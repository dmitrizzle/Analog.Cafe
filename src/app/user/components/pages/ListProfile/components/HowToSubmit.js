import React from "react"

import ArticleSection from "../../../../../core/components/pages/Article/components/ArticleSection"
import Link from "../../../../../core/components/controls/Link"

export default () => {
  return (
    <ArticleSection>
      <h3>Your submissions.</h3>
      {/* <p>
        You’re now a full member of Analog.Cafe community, nice!{" "}
        <span role="img" aria-label="OK sign with fingers">
          👌
        </span>
      </p> */}
      <p>
        If you love shooting film and have a story to share, Analog.Cafe could
        be the place to publish it. Submissions are now accepted for essays and
        articles on or featuring film photography.{" "}
        <Link to="/submit">Details</Link>.
      </p>
      {/* <p>
        Until then, your{" "}
        <Link to="/favourites">
          <strong>Favourites</strong>
        </Link>{" "}
        well . There you can store a private list of
        articles which you{" "}
        <span style={{ color: "#ed236e", fontWeight: "700" }}>❤︎</span>
        ’d.
      </p> */}
      {/* <p>
        Don’t forget to checkout the{" "}
        <Link to="/resources">
          <strong>Resources</strong>
        </Link>{" "}
        page, where you may find lots of useful guides, printables, and secret
        essays, not available to non-members.{" "}
        <span role="img" aria-label="Wink, wink">
          😉
        </span>
      </p> */}
    </ArticleSection>
  )
}
