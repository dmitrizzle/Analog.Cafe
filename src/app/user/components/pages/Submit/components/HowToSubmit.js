import React from "react"

import Figure from "../../../../../core/components/vignettes/Picture/components/Figure"
import Link from "../../../../../core/components/controls/Link"

export default () => {
  return (
    <div>
      <Figure
        src="https://res.cloudinary.com/analog-cafe/image/upload/v1528904759/image-froth_1010453_425a5704760c4879b31e008315c3047c.gif"
        alt="A short demo of the submission process to Analog.Cafe"
        caption
      >
        Analog.Cafe “Composer” tool is easy to use and works offline.
      </Figure>
      <p>
        <strong>Learn more</strong> about the types of content we publish,
        quality prerequisites, edits, and our mission in Dmitri’s editorial “
        <Link to="/zine/open-call-g99w">Open Call</Link>
        .”
      </p>
    </div>
  )
}
