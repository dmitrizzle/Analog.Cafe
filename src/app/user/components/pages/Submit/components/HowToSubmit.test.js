import { shallow } from "enzyme"
import React from "react"
import "jest-styled-components"

import HowToSubmit from "./HowToSubmit"

it("Render HowToSubmit, matches snapshot", () => {
  const element = shallow(<HowToSubmit />)
  expect(element).toMatchSnapshot()
})
