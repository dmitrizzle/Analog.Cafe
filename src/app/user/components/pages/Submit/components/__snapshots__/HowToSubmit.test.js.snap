// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render HowToSubmit, matches snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Unknown />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h3>
          How to submit.
</h3>,
        <Unknown
          alt="A short demo of the submission process to Analog.Cafe"
          src="https://res.cloudinary.com/analog-cafe/image/upload/v1528904759/image-froth_1010453_425a5704760c4879b31e008315c3047c.gif"
>
          The “Composer” tool will help you create and send submissions.
</Unknown>,
        <p>
          <em>
                    <strong>
                              First:
                    </strong>
          </em>
           
          go to
           
          <em>
                    <Unknown
                              to="/submit/compose"
                    >
                              Submit Now
                    </Unknown>
          </em>
           
          page and start creating your submission with text and images. As you edit, your work is saved automatically.
</p>,
        <p>
          <em>
                    <strong>
                              Then:
                    </strong>
          </em>
           
          make sure that your submission has at least one image: JPG or PNG. It must also include a title and text.
</p>,
        <p>
          <em>
                    <strong>
                              Finally:
                    </strong>
          </em>
           
          click “send!”
</p>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "How to submit.",
        },
        "ref": null,
        "rendered": "How to submit.",
        "type": "h3",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "alt": "A short demo of the submission process to Analog.Cafe",
          "children": "The “Composer” tool will help you create and send submissions.",
          "src": "https://res.cloudinary.com/analog-cafe/image/upload/v1528904759/image-froth_1010453_425a5704760c4879b31e008315c3047c.gif",
        },
        "ref": null,
        "rendered": "The “Composer” tool will help you create and send submissions.",
        "type": [Function],
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <em>
              <strong>
                            First:
              </strong>
</em>,
            " ",
            "go to",
            " ",
            <em>
              <Unknown
                            to="/submit/compose"
              >
                            Submit Now
              </Unknown>
</em>,
            " ",
            "page and start creating your submission with text and images. As you edit, your work is saved automatically.",
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <strong>
                First:
</strong>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "First:",
              },
              "ref": null,
              "rendered": "First:",
              "type": "strong",
            },
            "type": "em",
          },
          " ",
          "go to",
          " ",
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <Unknown
                to="/submit/compose"
>
                Submit Now
</Unknown>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "function",
              "props": Object {
                "children": "Submit Now",
                "to": "/submit/compose",
              },
              "ref": null,
              "rendered": "Submit Now",
              "type": [Function],
            },
            "type": "em",
          },
          " ",
          "page and start creating your submission with text and images. As you edit, your work is saved automatically.",
        ],
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <em>
              <strong>
                            Then:
              </strong>
</em>,
            " ",
            "make sure that your submission has at least one image: JPG or PNG. It must also include a title and text.",
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <strong>
                Then:
</strong>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Then:",
              },
              "ref": null,
              "rendered": "Then:",
              "type": "strong",
            },
            "type": "em",
          },
          " ",
          "make sure that your submission has at least one image: JPG or PNG. It must also include a title and text.",
        ],
        "type": "p",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <em>
              <strong>
                            Finally:
              </strong>
</em>,
            " ",
            "click “send!”",
          ],
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <strong>
                Finally:
</strong>,
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": "Finally:",
              },
              "ref": null,
              "rendered": "Finally:",
              "type": "strong",
            },
            "type": "em",
          },
          " ",
          "click “send!”",
        ],
        "type": "p",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h3>
            How to submit.
</h3>,
          <Unknown
            alt="A short demo of the submission process to Analog.Cafe"
            src="https://res.cloudinary.com/analog-cafe/image/upload/v1528904759/image-froth_1010453_425a5704760c4879b31e008315c3047c.gif"
>
            The “Composer” tool will help you create and send submissions.
</Unknown>,
          <p>
            <em>
                        <strong>
                                    First:
                        </strong>
            </em>
             
            go to
             
            <em>
                        <Unknown
                                    to="/submit/compose"
                        >
                                    Submit Now
                        </Unknown>
            </em>
             
            page and start creating your submission with text and images. As you edit, your work is saved automatically.
</p>,
          <p>
            <em>
                        <strong>
                                    Then:
                        </strong>
            </em>
             
            make sure that your submission has at least one image: JPG or PNG. It must also include a title and text.
</p>,
          <p>
            <em>
                        <strong>
                                    Finally:
                        </strong>
            </em>
             
            click “send!”
</p>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "How to submit.",
          },
          "ref": null,
          "rendered": "How to submit.",
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "function",
          "props": Object {
            "alt": "A short demo of the submission process to Analog.Cafe",
            "children": "The “Composer” tool will help you create and send submissions.",
            "src": "https://res.cloudinary.com/analog-cafe/image/upload/v1528904759/image-froth_1010453_425a5704760c4879b31e008315c3047c.gif",
          },
          "ref": null,
          "rendered": "The “Composer” tool will help you create and send submissions.",
          "type": [Function],
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <em>
                <strong>
                                First:
                </strong>
</em>,
              " ",
              "go to",
              " ",
              <em>
                <Unknown
                                to="/submit/compose"
                >
                                Submit Now
                </Unknown>
</em>,
              " ",
              "page and start creating your submission with text and images. As you edit, your work is saved automatically.",
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <strong>
                  First:
</strong>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "First:",
                },
                "ref": null,
                "rendered": "First:",
                "type": "strong",
              },
              "type": "em",
            },
            " ",
            "go to",
            " ",
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <Unknown
                  to="/submit/compose"
>
                  Submit Now
</Unknown>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "function",
                "props": Object {
                  "children": "Submit Now",
                  "to": "/submit/compose",
                },
                "ref": null,
                "rendered": "Submit Now",
                "type": [Function],
              },
              "type": "em",
            },
            " ",
            "page and start creating your submission with text and images. As you edit, your work is saved automatically.",
          ],
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <em>
                <strong>
                                Then:
                </strong>
</em>,
              " ",
              "make sure that your submission has at least one image: JPG or PNG. It must also include a title and text.",
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <strong>
                  Then:
</strong>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Then:",
                },
                "ref": null,
                "rendered": "Then:",
                "type": "strong",
              },
              "type": "em",
            },
            " ",
            "make sure that your submission has at least one image: JPG or PNG. It must also include a title and text.",
          ],
          "type": "p",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <em>
                <strong>
                                Finally:
                </strong>
</em>,
              " ",
              "click “send!”",
            ],
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <strong>
                  Finally:
</strong>,
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": "Finally:",
                },
                "ref": null,
                "rendered": "Finally:",
                "type": "strong",
              },
              "type": "em",
            },
            " ",
            "click “send!”",
          ],
          "type": "p",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
